### PR TITLE
PDF Upload, Zube 90: Shrine, Uppy, and S3 Configuration

### PR DESCRIPTION
**Pull request by ghost**
## Summary
Updated Rails to 5.2.3 🎉 . Also added `webpacker` 🐩 's 🎉 🎉 🎊 . Boom. 🎤 drop. 

Generated a migration on `locations` to add a column for `file_uploads`. Added that association to the location model (a location has many file uploads). Updated the location controller to permit the file uploads attributes. 

Added Uppy and Shrine to the project for the file uploads. 

**Zube Card Referenced** 90

**File Addition(s)** 
`application/javascript/packs/application.js`: The entry point for webpack
`application/javascript/fileUpload`: The configuration for uppy.

**Files Modified**
- `location.rb`: Added 'English' to the list of languages

**Libraries Added**
- `gem 'webpacker', '~> 5.1'`: Used to add webpack
- `yarn add uppy`

### Tests to Run: TODO
- `spec/features/whatver_spec.rb`: Tests the function `hello_world`

### TODO
- [ ] Commit test for file upload

### CODE REVIEW

File: app/models/file_upload.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class FileUpload < ActiveRecord::Base
+  include ImageUploader::Attachment(:image)
```
Comment (justuseapen) [inline app/models/file_upload.rb:2]: Is a pdf considered an image? Has this been tested with pdfs?

Comment (tinadev) [reply app/models/file_upload.rb:2]: Yes and yes

File: db/structure.sql [hunk 2]
```diff
@@ -1541,5 +1812,7 @@ INSERT INTO "schema_migrations" (version) VALUES
 ('20200621214426'),
 ('20200621222659'),
 ('20200621223318');
+('20200629173821'),
```
Comment (danivovich) [inline db/structure.sql:1815]: This is a bad merge it seems, the semicolon in the line above will not let these 2 additional versions be valid

Comment (tinadev) [reply db/structure.sql:1815]: Fixed. Thank you

**Review by justuseapen — APPROVED** [review]

**Pull request merged**